api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -282,7 +293,40 @@ public async Task<IActionResult> Preview(ReportModel model)
             instance.Sections = templateModel.Sections;
         });
 
-        return new JsonResult(new ReportPreviewModel(subject, body, results));
+        // Find all charts and make a request to the Charts API to generate the image.
+        await model.Sections.ForEachAsync(async section =>
```

Comment:
Report preview now includes the charts.  I need to move this logic into another location so that the ReportManager can use it too.